src/adapter_test_helpers/sumo_basic_test_helper.erl

```diff
@@ -195,4 +216,24 @@ init_store(Name) ->
     weird_field2  => [1, true, <<"hi">>, 1.1],
     weird_field3  => #{a => 1, b => [1, "2", <<"3">>], <<"c">> => false}
   })),
+
+  8 = sumo:count(Name),
+  _ = try sumo:count(wrong)
+  catch
+    _:no_workers -> ok
+  end,
+
   ok.
+
+%%%=============================================================================
+%%% Internal functions
+%%%=============================================================================
+
+pick_up_event(Event) ->
+  sumo_test_people_mnesia_events_handler:pick_up_event(Event).
```

Comment:
If the function is going to be used by every tester module, it shouldn't be in the mnesia one. Either the module should be called `sumo_test_people_event_handler` or the original one should be defined here and `sumo_test_people_mnesia_events_handler` should be the one calling this function, not the other way around.